atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -256,4 +257,13 @@ private String getShortOverflowTableName() {
             throw Throwables.propagate(e);
         }
     }
+
+    private String getPrimaryKeyConstraintName(String tableName) {
+        return truncateToMaxOracleLength("pk_" + tableName);
+    }
```

Comment:
We could probably extract this to a common utils class as well, it's used in several places.